app/domain/operations/eligible/migrate_eligibility.rb

```diff
@@ -40,10 +40,48 @@ def find_subject(values)
         Success(subject)
       end
 
+      def add_renewal_eligibilities(values)
+        prev_year = Date.today.year - 1
+        eligibility_years =
+          values[:current_eligibilities].collect(&:start_on).map(&:year).uniq
+        return unless eligibility_years == [prev_year]
```

Comment:
Here do you get multiple years or any single year?
If yes then you might need to check this way right?
```
eligibility_years.include?(prev_year)
```

Reply:
if you get multiple years then are you checking whether a single previous year is present in the array or do you want to see whether that previous year is included in that array? @raghuramg